Assets/Scripts/UI/MainMenu/MainMenuController.cs

```diff
@@ -89,6 +89,7 @@ private void Start()
         playerInputManagerController.AddJoinListener();
         playerInputManagerController.PlayerInputManager.splitScreen = false;
         playerInputManagerController.onPlayerInputJoined += AddPlayer;
+        ((Peer2PeerTransport)NetworkManager.singleton).OnPlayerRecieved += UpdateStartButton;
```

Comment:
This thing right here is the culprit (note that we should NOT have the MainMenuController active in the client lobby scene, ideally 🙃)